distribution/docs/src/main/resources/_contents/_sources/solr-catalog-provider-contents.adoc

```diff
@@ -66,30 +48,30 @@ a|* Does not scale. Only runs one single server instance.
 a|* Can be embedded in Java easily.
 * Requires no HTTP connection.
 * Uses the same interface as the Standalone Solr Server uses under the covers.
-* Allows for full control over the Solr Server. No synchronous issues on startup; i.e., the Solr Server will synchronously start up with the Solr Catalog Provider
-* Runs within the same JVM
+* Allows for full control over the Solr Server. No synchronous issues on startup; i.e., the Solr Server will synchronously start up with the Solr Catalog Provider.
+* Runs within the same JVM.
 * Setup and Installation is simple: unzip and run.
-|* Can only be interfaced using Java
+a|* Can only be interfaced using Java.
 
 |(Administrative)Tools
-a|* External open source tools like Luke will work to allow admins to check index contents
-* JMX metrics reporting is enabled
+a|* External open source tools like Luke will work to allow admins to check index contents.
+* JMX metrics reporting is enabled.
 
 a|* No Web Console. 
 * No easy way to natively access (out of the box) what is in the index files or health of server at the data store level.
 
 |Security
 a|* Does not open any ports which means no ports have to be secured.
-|
+a|
  
 |Performance
-a|* Requires no HTTP or network overhead
-* Near real-time indexing
-* Can understand complex queries
-|
+a|* Requires no HTTP or network overhead.
+* Near real-time indexing.
+* Can understand complex queries.
+a|
 
 |Backup/Recovery
-a|* Can manually or through custom scripts back up the indexes
-a|* Must copy files when server is shutdown
+a|* Can manually or through custom scripts back up the indexes.
```

Comment:
Should this say: "Backups of indexes can be preformed manually or through custom scripts"